File: HFSGuardaDiretorioVS2022/HFSGuardaDiretorio_MFC/hfsguardadir/objetos/DiretorioOrdem.cpp
#include "stdafx.h"
#include "DiretorioOrdem.h"

namespace hfsguardadir {

namespace objetos {

DiretorioOrdem::DiretorioOrdem()
{

}

int DiretorioOrdem::getOrdem() {
    return ordem;
}

void DiretorioOrdem::setOrdem(int ordem) {
    this->ordem = ordem;
}

}

}

File: HFSGuardaDiretorioVS2022/HFSGuardaDiretorio_VBNet/objetos/DiretorioOrdem.vb
'
' * Created by SharpDevelop.
' * User: henrique.souza
' * Date: 03/07/2015
' * Time: 14:47
' * 
' * To change this template use Tools | Options | Coding | Edit Standard Headers.
' 


Namespace objetos
	''' <summary>
	''' Description of DiretorioOrdem.
	''' </summary>
	Public Class DiretorioOrdem
		Private m_ordem As Integer

		Public Property Ordem() As Integer
			Get
				Return m_ordem
			End Get
			Set
				m_ordem = value
			End Set
		End Property
	End Class
End Namespace
